Core/src/Geometry/GeometryIdentifier.cpp

```diff
@@ -24,7 +24,7 @@ std::ostream& Acts::operator<<(std::ostream& os, Acts::GeometryIdentifier id) {
                                               id.sensitive(), id.extra()};
 
   bool writeSeparator = false;
-  for (auto i = 0u; i < 5u; ++i) {
+  for (auto i = 0u; i < (sizeof(levels) / sizeof(levels[0])); ++i) {
```

Comment:
That's not great. Can we use a `static const std::array` up top and use `size()` here?

Reply:
You're right, I overlooked this somehow. We should use `std::array` for both `names` and `levels`.